Readme.md

```diff
@@ -88,5 +88,5 @@ Our MIT license is designed to be as simple, and liberal as possible.
 [@mgreter]: https://github.com/mgreter
 [@xzyfer]: https://github.com/xzyfer
 
-sass2scss was originally written by [Marcel Greter](@mgreter)
+sass2scss was originally written by [Marcel Greter](https://github.com/mgreter)
```

Comment:
If i move the reference links to the end of the file, the link `[Marcel Greter](@mgreter)` still don't works. It only seams to work if i write it like this`[@mgreter]`.